EnglishFilter/sections/foreign.txt

```diff
@@ -6393,10 +6393,15 @@ soft98.ir#$#[id^="sidebar"][id*="stick"] > section { opacity: 0!important; point
 soft98.ir#$#.a4d2x { display: none!important; }
 soft98.ir#$#.d-js2-none { display: none!important; }
 soft98.ir#$#.d-js2-block { display: block!important; }
+soft98.ir##body > .toast
+soft98.ir##body > div[id][class] > div[class] > section.card
+soft98.ir###content-bottom__inner > aside#sidebar > section.bg-light
+soft98.ir##div[class^="col-12"] > div[id][class] > a > img
```

Comment:
`soft98.ir##body > .toast` - I don't see this.
`soft98.ir##body > div[id][class] > div[class] > section.card` - try, for example, `soft98.ir##body > div#content-top + div.container`
`soft98.ir##div[class^="col-12"] > div[id][class] > a > img` - I guess you block this image via URL, so you can remove it

Reply:
.toast is a popup that appears sometimes
![image](https://user-images.githubusercontent.com/68852737/92367112-ab07dc80-f0fe-11ea-98aa-58359de4ecf3.png)
"To view the link, disable the ad removal plugin in your browser"
Moved to annoyances.